apps/designer/app/designer/features/style-panel/shared/dependencies.ts

```diff
@@ -31,18 +31,26 @@ export const dependencies: Dependencies = {
   },
   flexContainers: {
     property: "display",
-    values: ["flex"],
+    values: ["flex", "inline-flex"],
   },
   // @todo this should actually check on parent
   flexItemsAndInFlowPseudos: {
     property: "display",
-    values: ["flex"],
+    values: ["flex", "inline-flex"],
   },
   // @todo needs to also check flex-wrap
   multilineFlexContainers: {
     property: "flexWrap",
     values: ["wrap", "wrap-reverse"],
   },
+  multiColumnElementsFlexContainersGridContainers: {
```

Comment:
do we need both or did you forget to delete this one?

Reply:
Just didn't touch existing in case we would want to show the gap only if a few children of grid/flex exist etc